package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "udaru",
-  "version": "3.2.0",
+  "version": "3.2.1",
```

Comment:
don't update version as we did not publish yet

Reply:
changed back to 3.2.0